helpers/flags.js

```diff
@@ -1 +1,16 @@
-export const getFlag = (key, defaultValue) => window.D2L?.LP?.Web?.UI?.Flags.Flag(key, defaultValue) ?? defaultValue;
+const flagOverrides = new Map();
+
+export function getFlag(key, defaultValue) {
+	if (flagOverrides.has(key)) {
+		return flagOverrides.get(key);
+	}
+	return window.D2L?.LP?.Web?.UI?.Flags.Flag(key, defaultValue) ?? defaultValue;
+}
+
+export function mockFlag(key, value) {
```

Comment:
Nice! I should update some of the tests I added for the list flag to use this.

Reply:
It seems to work well. The downside is you can't save the value at the top of a file, since that only gets evaluated when all the files first load, which happens before the tests can mock the value.